### PR TITLE
Add dotCSS/dotJS 2018 blogpost

### PR DESCRIPTION
**Pull request by gurtjun**

### CODE REVIEW

**Comment by yannickdeturck** [comment]
What's the status on this one? Are there still parts that will get added to the post?

File: _posts/2018-11-8-dot-css-dot-js-2018.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+When they published the State of JS results of 2018 there was a lot of commotion regarding the results.
+More specifically regarding the front end frameworks Angular, React and Vue.
+The State of JS survey declares that Angular is suffering alot in comparison with React and Vue.
+The Angular results are very disappointing and show us a high rate of dissatisfaction within its userbase.
+On the other side there is only love for both React and Vue.
+This lead to a war on Twitter and other channels saying the survey is flawed.
+
+Whether or not this is the case, I believe this survey can help lots of developers make choices and it gives at least some insights on the State of JavaScript.
+Besides the front end frameworks, the State of JS also offers lots of data of other subjects regarding JavaScript which are definitely worth checking out!
```
[the remark this answers is not in the document]
Comment (mdewree) [reply _posts/2018-11-8-dot-css-dot-js-2018.md]: Already added a few lines above.

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+1. 3-digit shorthand 
+2. Line graph
+3. Hue from shape 
+4. Lightness from total
+5. Saturation from range
+
+Just like this I bet you have no idea what these steps mean.
+But let us teach you the magic of reading color hex codes as well thanks to David.
+
+To get started he teached us that the best way to describe colors is with the HSL color model.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: teached -> taught

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+HSL stands for Hue, Saturation and Lightness. 
+Hue is the pure pigment of a color and can be described with 12 color names.
+We'll show you the color names later in Step 3.
+Saturation is how vibrant or muted the hue is and can be described as *saturated*, *washed*, *muted* or *gray*.
+Lightness speaks for it's self and describes how light or dark the color is and this can be done with *light*, *middle* or *dark*.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: itself*

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+
+Now you know how you can describe a color all you have to do is to follow the 5 steps.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: all you have to do is follow (de 'to' voor de follow weg)

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+And deterimine the Hue, lightness and saturation.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: Hue met kleine letter

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+With the shorthand from step one we have to create a linegraph based on the numbers of the hexcode.
+With basic understanding of hexadecimal numbers we can visualize a little line graph for the channel values.
+D is high, 9 is around the middle, 2 is low.
+And that's how we get our litle line graph.
+
+<img class="image fit" alt="Hex linegraph example" style="max-width: 25%;" src="/img/2018-11-08-dot-css-dot-js-2018/hex-linegraph.jpg">
+<img class="image fit" alt="Line graph of shorthand hex for color #D92" style="max-width: 25%;" src="/img/2018-11-08-dot-css-dot-js-2018/linegraph.jpg">
+
+### Step 3: Hue from shape
+This is what we think  it's the most tricky part about reading color hex codes.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: Spatie te veel tussen think en 'it's'. It's best ook veranderen naar 'is' in deze zin, oftewel: "This is in our opinion the most tricky part..."

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 7]
```diff
@@ -0,0 +1,292 @@
...
+With the line graph we got from the previous step you'll have to find a matching color on the color wheel.
+So you'll have to remember this one by hearth if you really want to show off ;)
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: heart* en punt (.) na de zin

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 8]
```diff
@@ -0,0 +1,292 @@
...
+For our example color it matches best with the color **orange**.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: Misschien komma na de eerste 'color' voor de leesbaarheid?

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 9]
```diff
@@ -0,0 +1,292 @@
...
+
+<img class="image fit" alt="HSL color wheel with color names" style="max-width: 75%" src="/img/2018-11-08-dot-css-dot-js-2018/color-wheel.jpg">
+
+### Step 4: Lightness from total
+To determine the lightness you should look at the total sum of the channel values (pairs).
+Or just look at the values in the line graph we created.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: Koppel deze zin misschien aan de vorige zin met een komma tussen of zo?

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 10]
```diff
@@ -0,0 +1,292 @@
...
+If the values are higher to the top, the color is closer to white and thus lighter.
+If the values are closer to the bottomn, the color is closer to black and thus darker.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: bottom*

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 11]
```diff
@@ -0,0 +1,292 @@
...
+For our example color `#D92`, the values are both high and low, so it has **middle** lightness.
+
+<img class="image fit" alt="Determining lightness of a color" style="max-width: 50%" src="/img/2018-11-08-dot-css-dot-js-2018/lightness.jpg">
+
+### Step 5: Saturation
+Saturation is a measure of how vibrant or rich the hue/color is.
+To measure the saturation whe need to see at the difference between the highest and the lowest value in our shorthand code.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: to look* at the difference

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 12]
```diff
@@ -0,0 +1,292 @@
...
+That's a wide range, but not completely wide.
+So our color has moderate saturation, thus making it a **washed** color.
+
+Now we have all three attributes for our color
+So we can say `#D49B25` is **Middle Washed Orange**.
+Pretty amazing right?
+
+<img class="image fit" alt="#D49B25 color described as Middle Washed Orange" style="max-width: 50%" src="/img/2018-11-08-dot-css-dot-js-2018/final-color.jpg">
+
+Since dotCss put's most of it's talks online we definitely recommend watching this talk.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: puts* en its*

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 13]
```diff
@@ -0,0 +1,292 @@
...
+He just explains it so well and goes a little bit deeper than we did here.  
+
+[View talk](https://www.dotconferences.com/2018/11/david-desandro-read-color-hex-codes){:target="_blank" rel="noopener noreferrer"} | [View slides](https://speakerdeck.com/desandro/read-color-hex-codes){:target="_blank" rel="noopener noreferrer"}
+
+## 'Variable fonts' by [Mandy Michael](https://twitter.com/mandy_kerr){:target="_blank" rel="noopener noreferrer"}
```
Comment (SamSchuddinck) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: Maybe give some credit to what she showed us?
She made some pretty nice animations which she showed. 

It think it's pitty we don't say anything about that.

File: _posts/2018-12-12-dot-css-dot-js-2018.md [hunk 14]
```diff
@@ -0,0 +1,292 @@
...
+As we all know, the JavaScript ecosystem is richer than ever, and even the most experienced developers become victims of the amount of choices they have to make.
+It's always changing. New libraries, frameworks, languagues...
+For this reason Sacha Greif, Raphaël Benitte and Michael Rambeau decided to create a global survey they called the 'State of JavaScript'.
+
+The survey contains data from over 20.000 developers from all around the globe and you can find the results on their [website](https://stateofjs.com){:target='_blank' rel='noopener noreferrer'}.
+The survey tries to figure out what these developers are using these days, what they like and what they would love to learn.
+
+When they published the State of JS results of 2018 there was a lot of commotion regarding the results.
+More specifically regarding the front end frameworks Angular, React and Vue.
+The State of JS survey declares that Angular is suffering alot in comparison with React and Vue.
```
Comment (Deryaa) [inline _posts/2018-12-12-dot-css-dot-js-2018.md]: a lot (spatie)

**Pull request merged**